docs/GettingStarted.md

```diff
@@ -65,16 +65,19 @@ const selector = ({ color = 'red' } = {}) => ({
 })
 
 // OR operator
-const selector = props => ({
-  color: props.color || 'red'
-})
+const selector = props => {
+  props = props || {}
```

Comment:
Ah, well I tried to get both examples on par, as the example above uses the default parameters and also sets a default object. The fact that Fela passes in an empty object is something we should note in the docs and then remove those default objects I think.